src/utils/match-path.js

```diff
@@ -0,0 +1,38 @@
+import { matchPath } from 'react-router-dom';
+import { routes } from '../config';
+
+export const findMatchedPath = (pathname, search) => {
+  // find match path
```

Comment:
I don't think we need this comment since the function name is very clear :)

Reply:
ah yeah sorry it wasnt meant to be there, will remove it